hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManager.java

```diff
@@ -115,9 +115,14 @@ NavigableSet<ContainerID> getContainersInPipeline(PipelineID pipelineID)
 
   void openPipeline(PipelineID pipelineId) throws IOException;
 
+  @Deprecated
```

Comment:
Why leave this deprecated on around? This is not an external facing API, so nothing should be using it except tests. It would probably be better to just remove it now.

Reply:
True, we can remove this method completely. I just didn't want to have large changes in a single PR if possible, it will become hard for the reviewers.

We will have to change another 15+ classes to remove this method, I will create a follow-up Jira to make this change.